byway/src/main/webapp/script.js

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Google LLC
+//
```

Comment:
Consider moving these functions into a separate js file, or renaming this file to be more specific to the page you're using them on.